pkgs/by-name/ap/apple-sdk/common/remove-disallowed-packages.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  jq,
+  stdenv,
+}:
+
+let
+  # This can be made unconditional once jq is available in the bootstrap tools. If corecrypto is  not removed from
+  # the umbrella framework, linking will fail in stage 1 because it can’t find the tbd.
```

Comment:
I’m not following. `disallowed-packages.json` contains globs of files belonging to packages to remove from the SDK. It has globs for corecrypto (because the text-based stubs should be removed), but because `jq` is not available in the bootstrap tools to process `libSystem.tbd` to remove the references, corecrypto has to be allowed in stage 1. Once the bootstrap tools are updated, corecrypto can be removed in every stage.